UI/Pages/Index.razor

```diff
@@ -7,4 +7,5 @@
         <a class="cursor-pointer mb-4 text-4xl bg-gradient-to-r from-pink-900 to-pink-500 w-60 text-center mx-auto block text-white p-3 rounded-r-full rounded-l-full">Start Game</a>
         <a class="cursor-pointer text-4xl bg-gradient-to-r from-orange-400 to-orange-500 mx-auto block text-white p-3 text-center w-60 rounded-r-full rounded-l-full">Join Game</a>
     </div>
-</div>
```

Comment:
Add a space after this line.

Reply:
The `Active` prop is set to `True`, yet the styling doesn't match what's on Figma for the Active state of a card. It needs a light border around the active card.